docs/reference/developer/adr/0009-handle-dimensionless-as-non-counts.rst

```diff
@@ -0,0 +1,38 @@
+ADR 0009: Handle dimensionless as non-counts
+============================================
+
+- Status: accepted
+- Deciders: Jan-Lukas, Neil, Owen, Simon
+- Date: 2021-09-01
+
+Context
+-------
+
+Repeatedly there is doubt over how dimensionless quantities should be handled in operations that deal with counts.
+This is primarily ``rebin`` and ``histogram``, but the effect is most visible in ``plot`` which internally rebins or resamples data.
+
+There is no "counts" unit in the SI base unit system but we have included "counts" in the scipp unit system from early on, since distinguishing dimensionless quantities from (neutron) counts is often essential.
```

Comment:
It might be useful for future readers to refer to a few examples of where behaviour would be incorrect or ill defined.

Reply:
See update.